examples/vg-specs/box-plot_minmax_2D_horizontal.vg.json

```diff
@@ -266,8 +266,7 @@
                     },
                     "yc": {
                         "scale": "y",
-                        "field": "age",
-                        "band": 0.5
```

Comment:
For context, it turns out that from scale merging, these plots are using point scale but the mark compiler previously read scale type from the original scale prior to merging.